packages/utils/src/lib/collect/index.ts

```diff
@@ -76,3 +83,29 @@ export async function collect(options: CollectOptions): Promise<Report> {
     }),
   };
 }
+
+export class ReadPackageJsonError extends Error {
+  constructor(outputPath: string) {
+    super(`outPath: ${outputPath} is no directory`);
+  }
```

Comment:
This is copy-pasted from elsewhere, it seems, doesn't fit here. I guess it should be something like this:

```suggestion
  constructor(message: string) {
    super(`error reading package.json: ${message}`);
  }
```